.flake8

```diff
@@ -6,5 +6,6 @@ max-complexity = 16
 select = C,E,F,W,B,B9
 # E203: whitespace before ':' (black behaviour)
 # E501: flake8 line length (covered by bugbear B950)
+# C901: function is too complex (out of our scope for now)  # todo
 # W503: line break before binary operator (black behaviour)
-ignore = E203,E501,W503
+ignore = E203,E501,C901,W503
```

Comment:
You should rather mute the warning locally with a `#  noqa`: [flake8-ignore-warnings-for-a-function](https://stackoverflow.com/questions/51127170/flake8-ignore-warnings-for-a-function)

Reply:
Thanks for the link, helped a lot!